ci-operator/step-registry/gather/must-gather/gather-must-gather-commands.sh

```diff
@@ -1,8 +1,7 @@
 #!/bin/bash
 
 set -o nounset
-set -o errexit
-set -o pipefail
+set +o errexit
```

Comment:
I'm not an approver, so no need to convince me that this is a good idea, but the softening here seems like it could cause us to not notice if must-gather starts failing.  Can we use more targeted `|| true` sections or something to soften the specific commands that need softening, or otherwise reduce the scope of allowed breakage?

Reply:
I could move it to the limited block I need it in instead, but errexit in must-gather always bothered me since I think if one thing fails it should still continue to try the rest (i.e. gather as much as possible).  When we get a partial must-gather which can happen, it's never collected because of the errexit.

Reply:
> ... I think if one thing fails it should still continue to try the rest (i.e. gather as much as possible).

I thought  the purpose of gather-extra was to cover when must-gather failed?

> When we get a partial must-gather which can happen, it's never collected because of the errexit.

What's not collected?

```sh
oc --insecure-skip-tls-verify adm must-gather $MUST_GATHER_IMAGE --dest-dir ${ARTIFACT_DIR}/must-gather > ${ARTIFACT_DIR}/must-gather/must-gather.log
```

means even a failing `must-gather` call will still get assets into `ARTIFACT_DIR` for collection, right?

Reply:
I guess, you still don't get camgi or the links. It's fine though, I can make the change more limited to where I need to remove errexit